ServerCore/Helpers/TeamHelper.cs

```diff
@@ -115,6 +115,11 @@ public static async Task DeleteTeamAsync(PuzzleServerContext context, Team team)
 
             context.TeamMembers.Add(Member);
             await context.SaveChangesAsync();
+
+            MailHelper.Singleton.SendPlaintextWithoutBcc(new string[] { team.PrimaryContactEmail, user.Email },
+                $"{user.Name} has now joined {team.Name}!",
+                $"Have a great time!");
```

Comment:
Something here should probably mention the event name to give the recipient some context?